NEWS.md

```diff
@@ -8,6 +8,7 @@
 * [MODSOURCE-690](https://issues.folio.org/browse/MODSOURCE-690) Make changes in SRS post processing handler to update MARC for shared Instance
 * [MODSOURCE-646](https://issues.folio.org/browse/MODSOURCE-646) Make changes to perform MARC To MARC Matching in Local Tenant & Central Tenant
 * [MODSOURCE-667](https://issues.folio.org/browse/MODSOURCE-667) Upgrade folio-kafka-wrapper to 3.0.0 version
+* [MODSOURCE-677](https://issues.folio.org/browse/MODSOURCE-677)  Import is completed with errors when control field that differs from 001 is used for marc-to-marc matching
```

Comment:
5.7.0 has been released already. This should be added in the new dev cycle log.
Please add the suggestion at the top of the file
```suggestion
## 2024-xx-xx 5.8.0-SNAPSHOT
* [MODSOURCE-677](https://issues.folio.org/browse/MODSOURCE-677)  Import is completed with errors when control field that differs from 001 is used for marc-to-marc matching
```

Reply:
Forgot to view the release version, thanks for noticing